src/cueUtilities.js

```diff
@@ -196,139 +220,193 @@ module.exports = function (params, cy, api) {
         node._private.data.expandcollapseRenderedStartX = expandcollapseStartX;
         node._private.data.expandcollapseRenderedStartY = expandcollapseStartY;
         node._private.data.expandcollapseRenderedCueSize = expandcollapseRectSize;
-        
-        nodeWithRenderedCue = node;
       }
 
       {
         cy.on('expandcollapse.clearvisualcue', function() {
 
-          if ( nodeWithRenderedCue ) {
+          if ( hoveredGroup ) {
             clearDraws();
           }
         });
 
-        cy.bind('zoom pan', data.eZoom = function () {
-          if ( nodeWithRenderedCue ) {
+        cy.on('zoom pan', data.eZoom = function (e) {
+          if (hoveredGroup || selectedGroups.length > 0) {
```

Comment:
Does the hovered cue need to be drawn here as well? You check for a hoveredGroup, but don't do anything with it.

Reply:
You're right, we don't need to check hovered cue.

Reply:
We actually do need it, and we should be drawing the hovered cue on zoom.

Reply:
Not sure you need this check any more now that both cases are more or less checked in refreshCanvasImages(). Maybe you're trying to prevent a needless redraw? If so, I guess that's ok.